starknet-rpc-test/Cargo.toml

```diff
@@ -26,9 +26,9 @@ thiserror = { workspace = true }
 tokio = { version = "1.34.0", features = ["rt", "macros", "parking_lot"] }
 url = "2.4.1"
 
-[[test]]
-name = "starknet_spec_version"
-path = "spec_version.rs"
+# [[test]]
```

Comment:
?

Reply:
This test is not ready and triggers clippy unused variables and so on. Better ignore it for noew